staging/kubeaddons-catalog/templates/service.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "kubeaddons-catalog.fullname" . }}
+  labels:
+{{ include "kubeaddons-catalog.labels" . | indent 4 }}
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 3000
```

Please use the port name from the pod definition:
```suggestion
    targetPort: http
```